TeXstudio: download dmg instead of zip

oculos opened the pull request:
New pull request so that TeXstudio is downloaded according to the new file structure of the project, where folders are named after the version.

hjuutilainen commented:
Hey @oculos, thank you for your efforts but can't merge these since I've just updated the TeXstudio recipes to use GitHub releases instead of parsing the SourceForge RSS feed. They seem to be using this from now on: https://github.com/texstudio-org/texstudio/releases

I've also updated the recipes for the disk image distribution.

I very much appreciate that you created these pull requests so keep them coming if you see something to improve!

oculos commented:
Cool, I'm happy to help, and nice the recipe will get the files directly
from the source!

On Mon, May 7, 2018 at 11:18 AM, Hannes Juutilainen <
notifications@github.com> wrote:

> Hey @oculos <https://github.com/oculos>, thank you for your efforts but
> can't merge these since I've just updated the TeXstudio recipes to use
> GitHub releases instead of parsing the SourceForge RSS feed. They seem to
> be using this from now on: https://github.com/texstudio-
> org/texstudio/releases
>
> I've also updated the recipes for the disk image distribution.
>
> I very much appreciate that you created these pull requests so keep them
> coming if you see something to improve!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/autopkg/hjuutilainen-recipes/pull/90#issuecomment-387008178>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGei80ZXmsGKTXvo7w6MIa6O9WN6O7ipks5twBFRgaJpZM4TyICX>
> .
>



-- 
Francis Augusto Medeiros
Oslo, Norway
+47 45171491
